gotfparse/pkg/converter/converter.go

```diff
@@ -107,22 +120,25 @@ func (t *terraformConverter) VisitJSON() *gabs.Container {
 			}
 
 			if len(obj) > 0 {
-				blockJSON.SetP(obj, arrayKey)
+				setP(b, blockJSON, "five", obj, arrayKey)
 			}
 		}
 
-		if jsonOut.ExistsP(arrayKey) {
```

Comment:
I think if the arrayKey already exists tfparse still wants to have the value, instead of merging, we append it as an array item. What was the thinking behind removing this step?

Reply:
There were some places where some resources were being doubled, when they should have been ovewritten. The use of the `dynamic` feature seems to produce three versions of each item. It's possible that this was caused by the issue above though. I'll see if the new fixes reveal that this was removed b/c of other bugs.